_includes/head.html

```diff
@@ -1,14 +1,12 @@
 <head>
-  <!-- Google Tag Manager -->
-  {%- if jekyll.environment == "production" -%}
-    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-    })(window,document,'script','dataLayer','GTM-NQRWDJK');</script>
-    <!-- End Google Tag Manager -->
-  {%- endif -%}
-
+ <!-- Matomo Tag Manager -->
+<script>
+var _mtm = window._mtm = window._mtm || [];
+_mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
+var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+g.async=true; g.src='https://cdn.matomo.cloud/rockarch.matomo.cloud/container_SdzKujX7_dev_cc9545323ac9afaa195e6699.js'; s.parentNode.insertBefore(g,s);
```

Comment:
```suggestion
g.async=true; g.src='https://cdn.matomo.cloud/rockarch.matomo.cloud/container_SdzKujX7.js'; s.parentNode.insertBefore(g,s);
```

Reply:
We can just use the live container environment. Happy to talk through our approach in the next sprint meeting.